### ISSUE TITLE
Documentation: 'typo' in softmax explanation image

### ISSUE THREAD

**Issue by alichaudry** [tensorflow/tensorflow#249]
In the top part of the attached image - is it just me or is the **X** matrix written the wrong way around? In keeping with dot product notation, shouldn't this be:

`y1 = W11 * x1 + W12 * x2 + W13 * x3 + b1`

and so on for y2 and y3?

![image](https://cloud.githubusercontent.com/assets/15057353/11194810/3a5bbad8-8c7c-11e5-869b-cd9e7b0e1b56.png)

**Comment by alichaudry** [comment]
I'm not sure if it's just me, but the image isn't rendering properly. Let me try to update it.

**Comment by vrv** [comment]
Don't worry, we've actually fixed it already in the repo.  We're trying to update the site with it soon.

https://github.com/tensorflow/tensorflow/issues/78 -- deduping
